pkg/api/aim2/controller/runs.go

```diff
@@ -105,6 +105,9 @@ func (c Controller) SearchRuns(ctx *fiber.Ctx) error {
 	if err = ctx.QueryParser(&req); err != nil {
 		return fiber.NewError(fiber.StatusUnprocessableEntity, err.Error())
 	}
+	if err := run.ValidateGetSearchRunsRequest(&req); err != nil {
```

Comment:
I'm thinking maybe we don't need this validation -- it's not an error, we should just craft the query so there are no results if there's no selected experiments

Reply:
i think we removed error in other cases where there's no selection, eg Metrics -- you just don't get results until you pick something.